jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/executor/PolicyExecutor.java

```diff
@@ -94,7 +93,7 @@ public PolicyExecutor (String jsonString) {
             source = jsonNode.get(PacmanSdkConstants.SOURCE).asText();
             targetType = jsonNode.get(PacmanSdkConstants.TARGET_TYPE).asText();
         } catch (Exception e) {
-            logger.error(DATA_ALERT_ERROR_STRING + JOB_NAME  + "error in getting Policy arguments  ->"+ e);
+            logger.error("error in getting Policy arguments -> " + jsonString, e);

```

Comment:
lets be consistent with notation please:

"... arguments -> {}", jsonString, e

Reply:
updated